src/proto/h1/conn.rs

```diff
@@ -475,25 +476,23 @@ where I: AsyncRead + AsyncWrite,
         }
     }
 
-    pub fn write_body(&mut self, chunk: Option<B>) -> StartSend<Option<B>, io::Error> {
+    pub fn write_body(&mut self, _cx: &mut task::Context, chunk: Option<B>) -> Poll<(), io::Error> {
```

Comment:
I think I prefer `cx` being the last argument, as it's "less important"... I filed https://github.com/rust-lang-nursery/futures-rs/issues/883 as well.

Reply:
Oh also, does this even need a `Context` argument? Seems it's not used?

Reply:
Oh yeah, in theory we could remove that one! I guess my thinking here was that since `Conn` is a public API, we might want to reserve the possibility of needing `Context` for future changes? Maybe that's a YAGNI though, so you tell me!